src/components/pages/ParentHome/ParentDashboard.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import Banner from '../../common/Banner';
 import ParentSidebar from './ParentSidebar';
+import ParentNavbar from './ParentNavbar';
 import '../../../styles/ParentStyles/dashboard.less';
 import { Layout } from 'antd';
 import 'antd/dist/antd.css';
 
 const ParentDashboard = () => {
+  const { Content } = Layout;
   return (
     <Layout>
       <ParentSidebar active="dashboard" />
-      <Banner />
+      <Content>
```

Comment:
As Tim has already stated, is the Content used here intended to be pulled from Layout on line 10?